docker/Dockerfile

```diff
@@ -21,7 +21,9 @@ FROM ${BASE_PYTHON_IMAGE}
 ARG SW_PYTHON_AGENT_PROTOCOL
 ARG SW_PYTHON_AGENT_VERSION
 
-RUN pip install --no-cache-dir "apache-skywalking[${SW_PYTHON_AGENT_PROTOCOL}]==${SW_PYTHON_AGENT_VERSION}"
+COPY dist/apache_skywalking*.whl /tmp/
```

Comment:
Is the `.whl` file is architecture independent? If it's architecture dependent we cannot simply copy the `.whl` file into the Docker image, instead we should build the `.whl` file inside Dockerfile. @Superskyyy any input?

Reply:
> Is the `.whl` file is architecture independent? If it's architecture dependent we cannot simply copy the `.whl` file into the Docker image, instead we should build the `.whl` file inside Dockerfile. @Superskyyy any input?

Our current `.whl` file is platform independent (`none`) and should be safe to copy (our codebase is currently in pure python), but let's not rely on this. I think we should go for the second option and build wheels inside dockerfile. 

(Btw if you try the python agent wheel on mac m1, it works right?)